tests/browser_test.py

```diff
@@ -65,6 +67,8 @@ async def test_review_signoff(
     )
     selenium.refresh()
 
+    time.sleep(1)  # give react a second for hooks
```

Comment:
https://github.com/mozilla/remote-settings/pull/512/files/01698db34da090fbf923df2a87ee9a2f2466b601..5b1fda47fccfad660d6971331a120faf622f1c93#diff-03ae4d94c3127061bcc8c5d44da8fe1745f8fe0ff6d3018e3cd95f6c697b3df6

Everything old is new again 😄 .

Same comment as there -- can we use Selenium to [explicitly wait](https://www.selenium.dev/documentation/webdriver/waits/#explicit-waits) instead of `time`? Either for a duration or [expected condition](https://www.selenium.dev/documentation/webdriver/support_features/expected_conditions/).

Reply:
Alright, that's annoying. Let's merge this to unblock ourselves, but can we make an issue to investigate this further?

Reply:
created https://github.com/mozilla/remote-settings/issues/522 to track it. We'll have to do the same thing in kinto-admin probably.